x/incentive/README.md

```diff
@@ -39,6 +40,27 @@ in collaboration with the `x/distribution`. For native stakers and validators,
 rewards are distributed through the `x/distribution` module after the initial
 distribution to BTC stakers and finality providers.
 
+### 1.1. Multi-Staking
+
+The incentive module supports multi-staking, which allows Bitcoin holders
+to delegate their BTC to multiple finality providers across different Babylon
+Shared Networks (BSNs) simultaneously. With multi-staking:
+
+- A single BTC delegation can secure multiple PoS chains simultaneously
+- Rewards are distributed proportionally across all finality providers
+  that the delegator is staking to
+- Each finality provider in a multi-staking setup can operate on different BSNs,
+  enabling cross-chain security provisioning
+- Reward calculations account for the delegator's voting power contribution
+  to each individual finality provider
+- Rewards handle multiple reward denominations simultaneously
```

Comment:
>- Rewards handle multiple reward denominations simultaneously

we were already handling that before :sweat_smile:, but it is your call to keep it